nsv13/code/modules/overmap/components.dm

```diff
@@ -373,7 +373,10 @@ Method to try locate an overmap object that we should attach to. Recursively cal
 
 /obj/structure/overmap/slowprocess()
 	. = ..()
-	try_repair(get_repair_efficiency() / 25) //Scale the value. If you have 80% of your armour plates repaired, the ship takes about 7.5 minutes to fully repair. If you only have 25% of your plates operational, it will take half an hour to fully repair the ship.
+	if(istype(src, /obj/structure/overmap/asteroid)) //Shouldn't be repairing over time
+		return
+	if(mass > MASS_TINY) //Prevents fighters regenerating
```

Comment:
Makes sense it prevents fighters from repairing.

Reply:
As per the change log, this is a bug fix. Fighters were never supposed to regenerate integrity.